bundles/org.openhab.binding.shelly/src/test/java/org/openhab/binding/shelly/internal/discovery/ShellyThingCreatorTest.java

```diff
@@ -174,6 +174,8 @@ private static Stream<Arguments> provideTestCasesForGetThingUIDReturnsThingUidBy
                 Arguments.of(SHELLYDT_PRO1PM, "", THING_TYPE_SHELLYPRO1PM_STR), //
                 Arguments.of(SHELLYDT_PRO1PM_2, "", THING_TYPE_SHELLYPRO1PM_STR), //
                 Arguments.of(SHELLYDT_PRO1PM_3, "", THING_TYPE_SHELLYPRO1PM_STR), //
+                Arguments.of(SHELLYDT_PRO1CB, "", THING_TYPE_SHELLYPRO1CB_STR), //
+                Arguments.of(THING_TYPE_SHELLYPRO1CB_STR, "", THING_TYPE_SHELLYPRO1CB_STR), //
```

Comment:
The purpose of the test using these parameters is to verify mapping by device type. If you want to verify the thing type mapping (self), I'd propose to add a new test for this.
```suggestion
```